doc/administrate.rst

```diff
@@ -96,18 +96,18 @@ The form to will ask for:
 - ``Open``: This is a checkbox to specify whether this election is already
   open for votes or not.
 
-- ``URL to claim a badge for voting``: this allows to specify a link where
-  people will be able to go to collect a badge announcing they participated
-  on this election. You should coordinate with the people of the
+- ``URL to claim a badge for voting``: This allows to specify a link where
+  people will be able to go to collect a badge announcing that they participated
+  in this election. You should coordinate with the people of the
   `badge <https://fedorahosted.org/fedora-badges/>`_ project to get this
   link.
 
-- ``Number of votes a user can make``: this specifies the number of choices
+- ``Number of votes a user can make``: This specifies the number of choices
   a user can make for this election.
-  For example, a user might be allowed to select only 16 wallpapers, thus
+  For example, if a user should be allowed to select only 16 wallpapers, then
```

Comment:
I'd replace ``should be allowed`` by ``is allowed``, the ``if`` added seems to make more sense with a present tense.